# Practical task repository for Error Handling

# Task Completed
--
* Using try, catch in function blocks in order to handle exceptions. 
--
